### DISCUSSION TITLE
Playtime leaderboards?

### DISCUSSION THREAD

**Discussion by ghost** [EssentialsX/Essentials#4639] [Ideas and Feature Suggestions] [answered]
Basically what the title states, could be a nice addition. In the same format as /baltop, from most playtime to least playtime.

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of Essentials as we do not store playtime and the ability to fetch the playtime statistics of offline players is limited to 1.15.2+. Another plugin should do this.
